From 4b2c6df4f59e948e57d45a947d384c862f17f93a Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 22 Jun 2020 09:53:55 -0400
Subject: [PATCH 03/19] Allow flags other than macros in CPPFLAGS

---
 CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9122a71e197..e248923817a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -687,7 +687,9 @@ ENDIF()
 # Following autotools tradition, add preprocessor definitions
 # specified in environment variable CPPFLAGS
 IF(DEFINED ENV{CPPFLAGS})
-  ADD_DEFINITIONS($ENV{CPPFLAGS})
+  SET(CPPFLAGS $ENV{CPPFLAGS})
+  SEPARATE_ARGUMENTS(CPPFLAGS)
+  ADD_COMPILE_OPTIONS(${CPPFLAGS})
 ENDIF()
 
 INCLUDE(CheckTypeSize)
-- 
2.33.1

