From f86d3a4460b8877b4d2bc27c211799bd64579502 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 19 Oct 2020 23:48:16 -0400
Subject: [PATCH 04/18] Add include header for std::string

---
 sql/create_def.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sql/create_def.cc b/sql/create_def.cc
index 33e58bac021..595ce98b5a5 100644
--- a/sql/create_def.cc
+++ b/sql/create_def.cc
@@ -68,6 +68,7 @@
 #include <thread>
 #include <unordered_set>
 #include <vector>
+#include <string>
 
 /** Prints an error message supplied and attaches GetLastError with formatted
  * message. */
-- 
2.33.1

