From 96388ca6b3ba508b0b9085f11c842544e2b2d2ab Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 19 Oct 2020 23:48:16 -0400
Subject: [PATCH 04/14] Add include header for std::string

---
 sql/create_def.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sql/create_def.cc b/sql/create_def.cc
index bd9a1658f0..644aac5029 100644
--- a/sql/create_def.cc
+++ b/sql/create_def.cc
@@ -64,6 +64,7 @@
 #include <thread>
 #include <unordered_set>
 #include <vector>
+#include <string>
 
 /** Prints an error message supplied and attaches GetLastError with formated
  * message. */
-- 
2.31.1

