From e72a6a1c2b57e7fda6fcb37d78c09393e0991f4e Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 19 Jun 2023 20:25:57 +0100
Subject: [PATCH 24/24] Make libfido checks conditional

---
 CMakeLists.txt | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7387743852..d655d2e923 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1928,8 +1928,10 @@ ENDIF()
 MYSQL_CHECK_RAPIDJSON()
 
 # Look for fido2 library
-MYSQL_CHECK_FIDO()
-MYSQL_CHECK_FIDO_DLLS()
+IF(WITH_AUTHENTICATION_CLIENT_PLUGINS)
+  MYSQL_CHECK_FIDO()
+  MYSQL_CHECK_FIDO_DLLS()
+ENDIF()
 
 IF(APPLE)
   GET_FILENAME_COMPONENT(HOMEBREW_BASE ${HOMEBREW_HOME} DIRECTORY)
-- 
2.40.1

